kili/cli.py

```diff
@@ -12,8 +14,70 @@ def cli():
     """Kili Command line interface."""
 
 
+@cli.group(context_settings=CONTEXT_SETTINGS)
+def project():
+    """Commands to interact with a Kili project"""
+
+
+@project.command(name='import')
+@click.argument('files', type=str, nargs=-1)
+@click.option('--api-key', type=str, help='Your Api Key', envvar='KILI_API_KEY')
+@click.option('--project-id', type=str, help='Id of the project to import assets in')
+@click.option('--exclude', '-e', type=str, multiple=True,
+              help="Files to exclude from the given files")
+@click.option('--frames', type=bool, help="Only for a frame project, import videos as frames")
+@click.option('--fps', type=int, help="Only for a frame project, import videos with this fps")
+#pylint: disable=too-many-arguments
+def import_assets(api_key, project_id, files, exclude, fps, frames):
+    """
+    Command for adding assets into a project.
+
+    Currently, this command does not support :
+        - The import of videos from local frames, rich text and time series assets.
+        - the import of assets with metadata or with a custom external_id
+    For such imports, please use the `append_many_to_dataset` method in the Kili SDK.
+    """
+    if files is None:
+        raise ValueError("No files or directory specified.")
+    kili = Kili(api_key=api_key)
+    input_type = kili.projects(project_id, disable_tqdm=True, fields=[
+                               'inputType'])[0]['inputType']
+
+    if input_type != 'FRAME' and (fps is not None or frames is not None):
+        illegal_option = 'fps and frames are'
+        if frames is None:
+            illegal_option = 'fps is'
+        if frames is None:
+            illegal_option = 'frames is'
+        raise ValueError(f'{illegal_option} only valid for a FRAME project')
+
+    files_to_upload = get_file_to_upload(files, input_type, exclude)
+    if len(files_to_upload) == 0:
+        raise ValueError(
+            'No files to upload.'
+            'Check that the paths exist and that the file types are compatible with the project')
+
+    external_ids = [path.split('/')[-1] for path in files_to_upload]
+
+    json_metadata_array = None
+    if input_type == 'FRAME':
+        json_metadata_array = [
+            {'processingParameters': {
+                'shouldKeepNativeFrameRate': fps is None,
+                'framesPlayedPerSecond': fps,
```

Comment:
fps can be None here

Reply:
This is wanted. If the fps option is not given, it will be None, then fps will be None and shouldKeepNativeFrameRate will be true. the video will be imported with its native fps.
Also, in `append_many_to_dataset`, if fps is none, it will be set to the default value which is 25

Reply:
ok